ergoCubSN000/hardware/motorControl/torso-eb5-j0_2-mc.xml

```diff
@@ -16,10 +16,10 @@
         <param name="jntPosMax">                23                  45                  43                  </param>
         <param name="jntPosMin">                -23                 -18                 -43                 </param>
         <param name="jntVelMax">                120                 120                 120                 </param>
-        <param name="motorNominalCurrents">     5000                15000               5000                </param>
-        <param name="motorPeakCurrents">        10000               20000               10000                </param>
-        <param name="motorOverloadCurrents">    15000               30000               15000               </param>
-        <param name="motorPwmLimit">            10000               16000               10000               </param>
+        <param name="motorNominalCurrents">     3770                3770                5000                </param>
+        <param name="motorPeakCurrents">        14000               14000               10000                </param>
+        <param name="motorOverloadCurrents">    18000               18000               15000               </param>
+        <param name="motorPwmLimit">            10000               10000               10000               </param>
```

Comment:
Why did we reduce also the PWM limit?
By contrast, I would tune these value to the full scale for both the joints. Being controlled in current, there's no harm in doing so.

To be checked the full scale here.

Reply:
@pattacini We reduced the pwm limit to the same level as iCub3. In addition, the roll joint (which has the same motor as the pitch) has a power limit of 10000.  So we made this choice.

However, should we increase it?

Reply:
yes, I agree, we can keep it at full scale.

Reply:
Let's go to the PWM full scale then!
We need to confirm its value first.

Reply:
Let's set them up to `16000`. Even if it might not be the full-scale, it's certainly higher than `10000` and conservative wrt our old thresholds.